firebase-firestore/firebase-firestore.gradle

```diff
@@ -94,6 +94,15 @@ android {
     testOptions.unitTests.includeAndroidResources = true
 }
 
```

Comment:
You can also add `ext.useProdBackendForTests = false` here, to make it the default if you like